playground/src/editor.tsx

```diff
@@ -56,9 +58,27 @@ function VSDiagsToMarkers(errors: VSDiagnostic[]): monaco.editor.IMarkerData[] {
   });
 }
 
+// get the language service profile from the URL
+// default to unrestricted if not specified
+export function getProfile(): TargetProfile {
+  const params = new URLSearchParams(window.location.search);
+  if (params.has("profile")) {
+    const profile = params.get("profile");
+    if (profile === "base") {
+      return "base";
+    }
+    if (profile === "quantinuum") {
+      return "quantinuum";
+    }
+  }
+
+  return "unrestricted";
```

Comment:
```suggestion
  return new URLSearchParams(window.location.search).get("profile") ?? "unrestricted";
```

Reply:
@billti This would allow arbitrary profile "names" to be set in the url, which violates the return type of TargetProfile. Is this something you'd like so that it 'just works' when we add new profiles? I can go either way as this is our playground.

Reply:
Assuming they code the code share link from the playground in the first place (where else?) I don't think we need to be overly cautious. Garbage in garbage out. (It may be equally confusing to 'think' you're passing a specific profile but it just keeps silently giving you 'unrestricted' behavior, as there is no error here).

I'm easy though. I wouldn't over think it :-)